### PR TITLE
Allowing CampaignPages to be viewed after having been created in core champaign

### PR DESCRIPTION
**Pull request by EricBoersma**

### CODE REVIEW

**Comment by EricBoersma** [comment]
@osahyoun Would you give this an eye and lend a thumbs up or some critique?

File: app/controllers/campaign_page_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CampaignPageController < ApplicationController
+  def view
+    @page = CampaignPage.find_by_slug params[:slug]
```
Comment (osahyoun) [inline app/controllers/campaign_page_controller.rb]: `CampaignPage.find_by! slug: params[:slug]`

The `!` will raise 404 if not found.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def not_found
```
Comment (osahyoun) [inline app/controllers/application_controller.rb]: You can drop this, as per below.

File: test/controllers/campaign_page_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (osahyoun) [inline test/controllers/campaign_page_controller_test.rb]: You've rails default testing framework. We should (in another PR) switch it with rspec.

**Comment by osahyoun** [comment]
:fireworks:

**Pull request merged**